pytest_watch/__main__.py

```diff
@@ -8,12 +8,14 @@
 :license: MIT, see LICENSE for more details.
 """
 
-import os
-import sys
 
+def run_cli():
+    import os
+    import sys
 
-if __name__ == '__main__':
     sys.path.append(os.path.dirname(__file__))
 
     from pytest_watch.command import main
     main()
+
+if __name__ == '__main__': run_cli()
```

Comment:
Oh, meant to say before. Could you move `run_cli` to its own line?

I generally find that more readable. [PEP8 also prefers it](https://www.python.org/dev/peps/pep-0008/#multiline-if-statements). (Search for "Compound statements (multiple statements on the same line) are generally discouraged".)

Reply:
done!